graphics/fonts/macfont.cpp

```diff
@@ -503,11 +503,14 @@ MacFONTFont *MacFONTFont::scaleFont(const MacFONTFont *src, int newSize, int sla
 		return NULL;
 	}
 
-	Graphics::Surface srcSurf, tmpSurf;
-	srcSurf.create(MAX(src->getFontSize() * 2, newSize * 2), MAX(src->getFontSize() * 2, newSize * 2),
-				PixelFormat::createFormatCLUT8());
-	int dstGraySize = newSize * 20 * newSize;
-	int *dstGray = (int *)malloc(dstGraySize * sizeof(int));
+	BdfFont* srcCopy = new BdfFont(src->_data, DisposeAfterUse::YES);
```

Comment:
Nad code formatting. And WHY BdfFont????

Reply:
Yes removed that, check the new commit